aspnet/fundamentals/servers.rst

```diff
@@ -125,42 +111,52 @@ The ``project.json`` file shown above demonstrates how to pass the ``server.urls
 
 .. literalinclude:: servers/sample/ServersDemo/src/ServersDemo/Startup.cs
 	:linenos:
-	:lines: 29-45
-	:emphasize-lines: 3,6-8,13
+	:lines: 37-54
+	:emphasize-lines: 3,6-7,10,15
 	:language: c#
 	:dedent: 8
 
+
 IIS and IIS Express
 -------------------
 
-Working with IIS as your server for your ASP.NET application is the default option, and should be familiar to ASP.NET developers who have worked with previous versions of the framework. IIS currently provides support for the largest number of features, and includes IIS management functionality and access to other IIS modules. Hosting ASP.NET 5 on IIS bypasses the legacy ``System.Web`` infrastructure used by prior versions of ASP.NET, providing a substantial performance gain. Most production ASP.NET web applications being deployed to Windows servers will run on IIS.
+Working with IIS as your server for your ASP.NET application is the default option, and should be familiar to ASP.NET developers who have worked with previous versions of the framework. IIS currently provides support for the largest number of features, and includes IIS management functionality and access to other IIS modules. Hosting ASP.NET 5 on IIS bypasses the legacy ``System.Web`` infrastructure used by prior versions of ASP.NET, providing a substantial performance gain.
+
+HTTPPlatformHandler
+^^^^^^^^^^^^^^^^^^^
+
+In ASP.NET 5, the web application is hosted by an external process outside of IIS. The HTTP Platform Handler is an IIS 7.5+ module which is responsible for process management of http listeners and to proxy requests to processes that it manages. 
+
+
 
 WebListener
 -----------
 
-WebListener is a Windows-only server that allows ASP.NET applications to be hosted outside of IIS. It runs directly on the `Http.Sys kernel driver <http://www.iis.net/learn/get-started/introduction-to-iis/introduction-to-iis-architecture>`_, and has very little overhead. It supports the same feature interfaces as IIS; in fact, you can think of WebListener as a library version of IIS.
+WebListener is a Windows-only application server for ASP.NET 5. It runs directly on the `Http.Sys kernel driver <http://www.iis.net/learn/get-started/introduction-to-iis/introduction-to-iis-architecture>`_, and has very little overhead. It supports the same feature interfaces as IIS; in fact, you can think of WebListener as a library version of IIS.
```

Comment:
remove the comment about feature interfaces, as those are no longer relevant to IIS.